monai/run_inference_single_image.py

```diff
@@ -72,6 +72,11 @@ def get_parser():
                         ' Default: 64x192x-1')
     parser.add_argument('--device', default="gpu", type=str, choices=["gpu", "cpu"],
                         help='Device to run inference on. Default: cpu')
+    parser.add_argument('--use-tta', action='store_true', 
+                        help='Use test-time augmentation (TTA), i.e. mirroring across all 3 axes. Default: False')
+    parser.add_argument('--remove-small-objects', type=int, default=10,
```

Comment:
Small comment:
The name of this flag made me thought that this was an "action" flag and not an "int". Maybe changing it to something like "small_object_size" or else, could make it more intuitive.

Reply:
yeah this is what I had some issues with. I initially intended it to be action flag but ended up letting the user decide the percent. 

The issue with `small_object_size` is that it doesn't have `remove` so it is not clear what it would do. How about `remove_smallest_x` ?

Reply:
in the end, kept the same arg but removed the default value. commit https://github.com/sct-pipeline/contrast-agnostic-softseg-spinalcord/pull/96/commits/39a71b839a8fd837e21bb8e1a21ea091aa4ba96c